definitions/reports.yml

```
@@ -672,16 +685,15 @@ components:
                         type: array
                         description: Records in JSON format
                         items:
-                          $ref: '#/components/schemas/csv_conversations_ip_voice'
-
+                          $ref: '#/components/schemas/csv_asr'
               - allOf:
                   - $ref: '#/components/schemas/report_response_top_level'
-                  - description: ASR
-                    x-tab-id: asr
+                  - description: CONVERSATIONS
```

```suggestion
                  - description: Conversations
```
This string shows on the render so it should be sentence case